horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/datastore/ElasticsearchDatastoreConfig.java

```diff
@@ -1,62 +1,44 @@
 package io.hyperfoil.tools.horreum.api.data.datastore;
 
-import static java.util.Objects.requireNonNullElse;
-
 import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import io.hyperfoil.tools.horreum.api.data.datastore.auth.APIKeyAuth;
+import io.hyperfoil.tools.horreum.api.data.datastore.auth.NoAuth;
+import io.hyperfoil.tools.horreum.api.data.datastore.auth.UsernamePassAuth;
 
 @Schema(type = SchemaType.OBJECT, required = true, description = "Type of backend datastore")
 public class ElasticsearchDatastoreConfig extends BaseDatastoreConfig {
 
+    public static final String[] auths = { NoAuth._TYPE, APIKeyAuth._TYPE, UsernamePassAuth._TYPE };
```

Comment:
```suggestion
```

Reply:
This is accessed via reflection: https://github.com/Hyperfoil/Horreum/pull/2223/files#diff-8a4bd7c5a2730d0b066069e4fe1d81fd44eb3c7f84b69867ad74b83f74aca8deR27-R30